library/aria.c

```diff
@@ -215,19 +215,19 @@ static inline void aria_sl( uint32_t *a, uint32_t *b,
     *a = ( (uint32_t) sa[ MBEDTLS_BYTE_0( *a ) ]       ) ^
          (((uint32_t) sb[ MBEDTLS_BYTE_1( *a ) ]) <<  8) ^
          (((uint32_t) sc[ MBEDTLS_BYTE_2( *a ) ]) << 16) ^
-         (((uint32_t) sd[ *a >> 24             ]) << 24);
+         (((uint32_t) sd[ MBEDTLS_BYTE_3( *a ) ]) << 24);
```

Comment:
Those changes should not be in the "Improve common.h macro documentation " commit as it is not an improvement of macro documentation.